src/internal/operators/retry.ts

```diff
@@ -56,6 +73,14 @@ export interface RetryConfig {
  * source stream when the source stream errors, at most `count` times.
  */
 export function retry<T>(count?: number): MonoTypeOperatorFunction<T>;
+
+/**
+ * A more configurable means of retrying a source.
+ *
+ * If `delay` is provided as a `number`, after the source errors, the result will wait `delay` milliseconds,
+ * then retry the source. If `delay` is a function, th
```

Comment:
Seems like this sentence is not finished.

Reply:
This is so weird. I feel like this is the third or forth time this has happened to me, but only with doc comments :\